### PR TITLE
Allow sorc/build_tropcy_NEMS.sh to work in Release/gfs.v16.3.8

### PR DESCRIPTION
**Pull request by ADCollard**
For some reason the sorc/build_tropcy_NEMS.sh no longer works on WCOSS2 machines.   @RussTreadon-NOAA and @aerorahul found fixes that allow the build to complete.  This PR implements one of these fixes - removing the references to the NEMSIOGFS module, which is being loaded but does not have a version number attached.

Six executables result from this build as expected.   They should still be tested for functionality.

This references #1812.

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Review by WalterKolczynski-NOAA — APPROVED** [review]

File: docs/Release_Notes.gfs.v16.3.8.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ This change turns of ingest of NOAA-21 radiances into the GSI until CRTM fix fil
 
 Also included in this update is a fix to a compilation error with "-check all" for gfs_bufrsnd, which resolves NCO bugzilla #1208.
 
+Remove NEMSIOGFS dependency in sorc/build_tropcy_NEWS.sh.
```
Comment (KateFriedman-NOAA) [inline docs/Release_Notes.gfs.v16.3.8.md]: Small spelling mistake in name of build script. Change "W" to "M".

File: docs/Release_Notes.gfs.v16.3.8.md [hunk 2]
```diff
@@ -74,6 +76,7 @@ SCRIPT CHANGES
 --------------
 
 * Changes to sorc/gsi.fd/scripts/exglobal_atmos_analysis.sh and sorc/checkout.sh
+* Change to sorc/build_tropcy_NEWS.sh
```
Comment (KateFriedman-NOAA) [inline docs/Release_Notes.gfs.v16.3.8.md]: Small spelling mistake in name of build script. Change "W" to "M".

**Review by KateFriedman-NOAA — APPROVED** [review]
Approve pending fixing the small spelling mistakes I noted. Thanks!

**Review by KateFriedman-NOAA — APPROVED** [review]

**Review by WalterKolczynski-NOAA — APPROVED** [review]

File: docs/Release_Notes.gfs.v16.3.8.md (hunk 2 above)
Comment (KateFriedman-NOAA) [reply docs/Release_Notes.gfs.v16.3.8.md]: Fixed

File: docs/Release_Notes.gfs.v16.3.8.md (hunk 1 above)
Comment (KateFriedman-NOAA) [reply docs/Release_Notes.gfs.v16.3.8.md]: Fixed

**Pull request merged**